packages/jest-expect/__typetests__/jest-expect.test.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {expectError, expectType} from 'tsd-lite';
+import {jestExpect} from '@jest/expect';
+import {expect} from 'expect';
+
+expectType<void>(jestExpect({}).toMatchSnapshot());
+
+expectError(() => {
+  expect({}).toMatchSnapshot();
+});
```

Comment:
Good one. In this case `jestExpect` and `expect` is an instance of the same object. So `expect` should have `.toMatchSnapshot()` in this test. Or?

Reply:
No it shouldn't, I think this is correct. While it will be there at runtime due to the unfortunate way `expect` does state, it shouldn't be in the types. So this is exactly what I want to happen 🙂

Reply:
If I get your suggestion right, `jestExpect` should be a clone of `expect` somehow? Or better to extend `Matchers` interface through module augmentation?

Reply:
I see your other comment, they both share state and globals. Cloning might go messy.

Reply:
yes. so at runtime they're the same, but the types should reflect the ideal reality of them being separate 🙂

Reply:
Alright. Sounds good (;